Formula/readosm.rb

```diff
@@ -35,18 +35,19 @@ def install
         doc/"examples/test_osm1",
         doc/"examples/test_osm2",
         doc/"examples/test_osm3",
-        doc/"examples/Makefile",
       ]
 
       shim_files.each do |f|
-        inreplace f, Superenv.shims_path, ""
+        inreplace f, Superenv.shims_path, HOMEBREW_PREFIX/bin
       end
+
+      inreplace doc/"examples/Makefile", "#{Superenv.shims_path}/", ""
```

Comment:
I'm not convinced we should even be shipping this file. I think we only should have the `*.c` files and maybe the docs for it to be honest.

Reply:
I changed this to delete the Makefiles and `*.o` files in the `examples` folder.  Now it only contains a Doxygen doc, `*.c` files, and shell scripts.